rpc/gnmi/gnmi-specification.md

```diff
@@ -1,76 +1,83 @@
 # gRPC Network Management Interface (gNMI)
 
 **Contributors:**
-Paul Borman, Marcus Hines, Carl Lebsack, Chris Morrow, Anees Shaikh, Rob Shakir, Wen Bo Li
+Paul Borman, Marcus Hines, Carl Lebsack, Chris Morrow, Anees Shaikh, Rob Shakir, Wen Bo Li, Darren Loher
 
 **Date:**
-April 18, 2023
+May 25, 2023
 
 **Version:**
-0.8.3
+0.10.0
```

Comment:
Is there anything that changed in 0.9.0 that we didn't cover in this update? Can we just file an issue if there is so that we don't lose track of it.

Reply:
Version 0.9.0 appears to be all whitespace changes.   Shouldn't have bumped the minor for that.   Ref:
 https://github.com/openconfig/gnmi/commit/d5360e33fc3b22effeaaaf55f345c1f950765bed#diff-49fc5824be3cfb8d64321cc64142f0568a3039a55da00d2aaed8fd766369854c